src/components/Modal/story.js

```diff
@@ -349,3 +350,26 @@ storiesOf('Next/Modal', module)
   .add('Nested Modals', () => {
     return <NestedModalDemo />
   })
+  .add('With Top Margin', () => {
+    return (
+      <Modal
```

Comment:
Let's wrap this in a provider and just pass `theme={{height: 2em}}` without importing the whole theme.